app/models/solarcycle.rb

```diff
@@ -1,4 +1,6 @@
 class Solarcycle < ApplicationRecord
   self.inheritance_column = :_type_disabled
   belongs_to :location
+
+  scope :after_beginning_of_yesterday, -> { where('created_at < ?', 1.day.ago.beginning_of_day) }
```

Comment:
nit but the function name is a little confusing to me? It could be a little less wordy

Reply:
Hrm. My concern was `after_yesterday` may not suggest we want `midnight` of yesterday rather than current time yesterday.